src/visualizers-workbench/P5VisualizerTemplate.ts

```diff
@@ -0,0 +1,174 @@
+import {P5Visualizer} from '../visualizers/P5Visualizer'
+import {VisualizerExportModule} from '@/visualizers/VisualizerInterface'
+
+/** md
+# p5 Visualizer Template
+
+[image should go here]
+
+This template can be used as a starting point for designing a p5 visualizer. You
+can use it to step back and forth through a sequence, showing each element as a
+written number.
```

Comment:
Yes, this is better, but it is now sort of mixing the meta-information about how this is a template with the example of how a p5 visualizer would document itself. That seems a little confusing. I can think of three possibilities: (1) put guidance comments *not* in markdown comments but in regular comments, so that they don't come through in the doc page for the example visualizer, and make those guidance comments typographically distinct and explain to the reader the convention; (2) if you do want the guidance to come through in the doc page, put all of the guidance in a markdown section at the top clearly indicating its purpose, and indicate where it switches over to straight-up example; or (3) have this file be _only_ straight-up example and have all of the commentary be in the separate "writing a visualizer" doc page. There may be other possibilities; fine with whatever you think makes the most sense, that keeps the example and the commentary about the example clearly delineated.

Reply:
Ok, that seems clear enough now.